### DISCUSSION TITLE
Κριτήρια Αξιολόγησης για την Επαναληπτική Εξεταστική του Σεπτεμβρίου

### DISCUSSION THREAD

**Discussion by IsabellaPap** [courses-ionio/sw#1568] [Q&A] [answered]
Καλησπέρα!

Θα ήθελα να ζητήσω κάποιες διευκρινίσεις αναφορικά με την εκπρόθεσμη αποστολή της τελικής αναφοράς κατά την Επαναληπτική εξέταση του Σεπτεμβρίου, η οποία δίνει την δυνατότητα σε φοιτητές που για **οποιοδήποτε λόγο** δεν κατάφεραν να περάσουν ή ακόμα και να παρευρεθούν σε κάποιο μάθημα του χειμερινού ή εαρινού εξαμήνου, να το κάνουν πριν την λήξη του ακαδημαϊκού έτους. Σύμφωνα με τις [οδηγίες](https://epidrome.github.io/teaching/faq/)
> H επαναληπτική εξεταστική (όπως λέει και το όνομα της άλλωστε) αφορά όσους προσπάθησαν και δεν τα κατάφεραν κατά την κανονική περιόδο.

Ωστόσο εγώ πιστεύω πως το "επαναληπτική" αναφέρεται στην διαδικασία της εξέτασης και όχι στην προσπάθεια των φοιτητών. Θεωρώ πως ο κύριος σκοπός είναι η ευελιξία που προσφέρουν στους φοιτητές, οι οποίοι, διαφέρουν από τους μαθητές προηγούμενων βαθμίδων στο γεγονός ότι συχνά αντιμετωπίζουν περισσότερες υποχρεώσεις πέρα από την φοίτηση τους.

Συγκεκριμένα χρειάζομαι διευκρινίσεις για το παρακάτω απόσπασμα από τις [οδηγίες](https://epidrome.github.io/teaching/faq/).

>Οι εργασίες της επαναληπτικής εξεταστικής παραδίδονται σύμφωνα με τις οδηγίες της αντίστοιχης χρονιάς (αν έχουν αλλάξει θα τις βρείτε στο ιστορικό του αρχείου readme) με προθεσμία την ημέρα της γραπτής εξέτασης, αλλά αν δεν έχουν ενδιάμεσα παραδοτέα κατά την κανονική περίοδο του μαθήματος βαθμολογούνται με μειωμένο βαθμό, εκτός και αν έχει γίνει πολύ σημαντική δουλειά πέρα από τα ζητούμενα, έτσι ώστε να μπορεί να δικαιολογηθεί ο πρόσθετος χρόνος εκπόνησης. Για πλήρη βαθμολόγηση της εργασίας θα πρέπει να ακολουθήσετε την διαδικασία των παραδοτέων από την αρχή μόνο στο αντίστοιχο εξάμηνο.

Οι διευκρινίσεις που ζητώ αφορούν:
- Τον καλύτερο ορισμό της "μειωμένης βαθμολογίας". Με βάση προηγούμενη [ανακοίνωση](https://github.com/courses-ionio/sw/discussions/1127) αλλά και τα λεγόμενα του κ. Χωριανόπουλου σε προηγούμενες συναντήσεις, η άριστη βαθμολογία φαίνεται να είναι το 5 για την εξεταστική του Σεπτεμβρίου. Θεωρώ πως μία "ποινή" ύψους 50% του βαθμού (δεδομένου πως το 50% είναι και η βάση για την ολοκλήρωση του μαθήματος) θα έπρεπε να είναι τουλάχιστον διακριτή αν όχι συζητήσιμη. Καταλαβαίνω πως η βαθμολογία δεν είναι πάντα arbitrary, αλλά για την απόκτηση του πτυχίου δυστυχώς ή ευτυχώς αυτό είναι το σύστημα που έχει οριστεί. 
- Τι συμβαίνει σε **ιδιαίτερες περιπτώσεις** όπου ένας φοιτητής ή μία φοιτήτρια δεν μπόρεσε να παρακολουθήσει το μάθημα κατά το εαρινό εξάμηνο εκείνου του έτους (στην δική μου περίπτωση έχω κάνει την προσπάθεια μου σε προηγούμενο έτος) άρα πρόκειται να **ξεκινήσει** την αναφορά **εντελώς εκπρόθεσμα**. Δικαιολογείται η εφαρμογή κανόνων που δυσκολεύουν τόσο πολύ την επιτυχία, απλώς επειδή κάποιος δεν ήταν διαθέσιμος για 3 μήνες; 

Στο σύνολο της λειτουργίας των πανεπιστημίων στην Ελλάδα φαίνεται να υπάρχει περισσότερη ευελιξία σε τέτοιες ιδιαίτερες περιπτώσεις από ότι στο μάθημα αυτό. Για κάποιον που ξεκινάει τον Ιούνιο το χρονικό περιθώριο είναι ίδιο με αυτό που είχαν όσοι το παρακολούθησαν κατα το εαρινό εξάμηνο, και η δυσκολία είναι ήδη μεγαλύτερη καθώς δεν υπάρχει η ίδια κινητικότητα στα forum (με την προϋπόθεση πως είναι λιγότεροι οι φοιτητές που επιδιώκουν να περάσουν κατά την επαναληπτική περίοδο). Είναι εύκολο να διαπιστωθεί πότε ξεκίνησε η συγγραφή της αναφοράς και δεν νομίζω πως κάποιος θα χρειαζόταν 2 εξάμηνα για να την ολοκληρώσει, οπότε φτάνω στο συμπέρασμα πως η υποβολή της αναφοράς τον Σεπτέμβριο σημαίνει απλά πως κάποιος δεν κατάφερε να δώσει την απαραίτητη προσοχή στο συγκεκριμένο μάθημα κατά το εαρινό εξάμηνο. Πιστεύω πως σε αυτή την περίπτωση οι ποινές που αφορούν εκπρόθεσμη υποβολή δεν θα έπρεπε να μεταφέρονται στην επαναληπτική εξεταστική. Δεν προτείνω η άριστη βαθμολογία να είναι το 10 χωρίς καμία ποινή, αλλά με άριστη βαθμολογία να είναι το 8 ακόμα και κατά την "κανονική" περίοδο, δεν θεωρώ πως υπάρχει και μεγάλο περιθώριο για περαιτέρω ποινές, πρέπει να υπάρξει μία κανονικοποίηση.

Προσωπικά είμαι διατεθειμένη να δώσω αναφορά σε contributions που έχω κάνει σε οpen source project, σε έρευνα πάνω στα θέματα που συζητήθηκαν κατά τη διάρκεια του μαθήματος, και σε ό,τι άλλο θεωρείται ωφέλιμο, αρκεί να είναι ξεκάθαρος ο τρόπος με το οποίον αυτά εκτιμούνται και να δοθεί περισσότερη σημασία στο νόημα και όχι στις προθεσμίες. Αυτό θα το βρείτε και στα [Συμπεράσματα της αναφοράς μου για το 2022](https://github.com/IsabellaPap/sw/tree/2019235/projects/2019235). Δεν έχει αλλάξει τίποτα.

Θα χαρώ να ακούσω τη γνώμη και άλλων φοιτητών πάνω στο θέμα καθώς πιστεύω πως και άλλοι βρίσκονται σε παρόμοια κατάσταση και ίσως έχουν εναλλακτικές λύσεις στο παραπάνω πρόβλημα.

**Comment by epidrome** [comment]
> εγώ πιστεύω πως το "επαναληπτική" αναφέρεται στην διαδικασία της εξέτασης και όχι στην προσπάθεια των φοιτητών. 

Τα μαθήματα δεν έχουν «εξέταση» που είναι μια έννοια παρωχημένη, αλλά συνεχή αξιολόγηση και ανατροφοδότηση της προσπάθειας, επομένως το μόνο πράγμα που μπορεί να επαναληφθεί μια προσπάθεια που έχει γίνει και αξιολογηθεί ήδη.

> Στο σύνολο της λειτουργίας των πανεπιστημίων στην Ελλάδα φαίνεται να υπάρχει περισσότερη ευελιξία σε τέτοιες ιδιαίτερες περιπτώσεις από ότι στο μάθημα αυτό.

Το γεγονός ότι μια διδακτική μέθοδος είναι δημοφιλής και οικεία δεν σημαίνει ότι είναι και σωστή-σύγχρονη.

Για μια σύντομη περιγραφή της φιλοσοφίας των μαθημάτων: https://epidrome.github.io/teaching/system/

**Reply by IsabellaPap** [reply]
Την έχω διαβάσει την φιλοσοφία του μαθήματος. Δεν διαφωνώ με αυτήν ούτε είναι ο σκοπός μου να συγκρίνω. Απαντήσατε πολύ επιλεκτικά στο θέμα μου (η μάλλον δεν απαντήσατε καθόλου σε αυτά που ήταν όντως ερωτήσεις και όχι δηλώσεις της άποψης μου). Δεν είστε ιδιαίτερα βοηθητικός σε αυτή την περίπτωση και δεν πιστεύω πως το issue μου είναι answered.

**Reply by epidrome** [reply]
Κατανοώ την διαφωνία σου και την σέβομαι, ελπίζω αυτό να είναι αμοιβαίο για την δικιά μου τοποθέτηση

Γιατί δεν γράφεις εσύ μια απάντηση που θεωρείς σωστή, άλλωστε ο τρόπος που είναι γραμμένη η ερώτηση σου είναι σαν να ξέρεις ήδη την απάντηση, βασικά σαν να πηγαίνεις στον γιατρό και να του δίνεις εσύ την γνωμάτευση και θεραπεία που πρέπει να εφαρμόσεις.

Για αυτό απάντησα στοχευμένα και όχι επιλεκτικά στις παραδοχές που έχεις κάνει και που κατά την γνώμη μου δεν είναι απλά λάθος, αλλά βασίζονται σε μια παρωχημένη εμπειρία εκπαίδευσης για την οποία φυσικά δεν φταις εσύ.

Εσύ θεωρείς ότι ένα μάθημα είναι απλά ένα σύνολο γνώσεων και δεξιοτήτων και στο τέλος αποκτάς μια πιστοποίηση π.χ., όπως ένα τέστ οδήγησης, ενώ εγώ θεωρώ πως ένα πανεπιστημιακό μάθημα είναι κυρίως ένας τρόπος σκέψης πάνω σε ένα γνωστικό αντικείμενο και μια προσπάθεια προσωπικού μετασχηματισμού και όχι απλά αναπαραγωγής (παπαγαλίας?) γνώσεων και δεξιοτήτων.

Εκτιμώ ότι τελικά δεν θα συμφωνήσουμε γιατί έχουμε διαφορετικά αξιακά συστήματα, αλλά ευχαρίστως να διαλέξω ως σωστή την απάντηση σου ως μια άποψη

**Reply by IsabellaPap** [reply]
Νομίζω πως έχω παρεξηγηθεί. Όπως ανέφερα πιο πάνω, δεν έχω κάποια διαφωνία στην φιλοσοφία του μαθήματος, ούτε θεωρώ κάτι από τα παραπάνω. Δεν διεκδικώ το πτυχίο της σχολής ως πιστοποίηση, απλώς δεν έχω την πολυτέλεια να περιμένω όσα χρόνια χρειαστεί για να ευθυγραμμιστεί η οικονομική και κοινωνική μου κατάσταση με τις απαιτήσεις του μαθήματος. Δεν είναι σκοπός μου να φιλοσοφήσω επί της ουσίας της εκπαίδευσης (ωστόσο δεν αρνούμαι πως το θέμα με ενδιαφέρει, και όπως ανέφερα παραπάνω, έχω διαβάσει την φιλοσοφία του μαθήματος που έχετε γράψει). Αν θυμάστε καθόλου από πέρυσι όπου **παρευρέθηκα σε όλες τις ομαδικές συναντήσεις** είχαμε κάνει πολύ εποικοδομητικό διάλογο πάνω σε αυτήν την φιλοσοφία και είχα μάλιστα επιζητήσει και περισσότερες πηγές τις οποίες και μου δώσατε για περαιτέρω ανάγνωση στο θέμα. 

Ωστόσο δυστυχώς όπως σας είχα αναφέρει και τότε ήταν μία χρονιά με μεγάλο φόρτο εργασίας για εμένα και ο λόγος που απέτυχα στο μάθημά σας ήταν πως δεν είχα την δυνατότητα να αφιερώνω 4-5 ώρες την εβδομάδα (χωρίς να μετράω τις δια ζώσης συναντήσεις) για τα ζητούμενα του μαθήματός σας. Αυτό ως εξήγηση για τα αυθαίρετα συμπεράσματα.

Τώρα για τις απορίες μου, γνωρίζετε πως παρότι ίσως δεν σας ταιριάζει, λειτουργείτε μέσα στο σύστημα που σας διατίθεται από το ελληνικό σύστημα εκπαίδευσης με αποτέλεσμα, να δημιουργείται ένα mix μεταξύ της φιλοσοφίας σας και του υπάρχοντος συστήματος, που κατά την δική μου άποψη δεν λειτουργεί προς όφελος ούτε της, κατά τα άλλα ενδιαφέρουσας, φιλοσοφίας του μαθήματος αλλά ούτε και των φοιτητών (γεγονός που αποδεικνύεται από κάποια στατιστικά επιτυχίας + τα μέτρα που πάρθηκαν φέτος για την διχοτόμηση του μαθήματος σας σε 2 μέρη με άλλο καθηγητή στην επίβλεψη).

Η απορία μου παραμένει η εξής: Γιατί είναι τόσο σημαντικοί παράγοντες οι (α) δια ζώσης παρουσία, (β) τήρηση εβδομαδιαίων προθεσμιών και (γ) να μην υπάρχει δυνατότητα να αφιερώσει κάποιος χρόνο το καλοκαίρι για τον Σεπτέμβριο χωρίς σημαντική μείωση του βαθμού (η οποία παραμένει μη ορισμένη πράγμα που δεν είναι πολύ διαφανές εκ μέρους σας). Αν μπορείτε να τα συνδυάσετε αυτά με την φιλοσοφία του μαθήματος τότε και εγώ θα δεχτώ το μέγιστο 5 που μπορώ να πάρω μέσω του κ. Βραχάτη και δεν θα σας ξαναενοχλήσω. Δεν θέλω να απαντήσω τις απορίες μόνη μου. Αν το ήθελα δεν θα τις ανέβαζα στο Q&A.

**Reply by KonstantinosTourtsakis** [reply, marked as answer]
α) Η διά ζώσης παρουσία είναι μέρος της ανάπτυξης λογισμικού για feedback και συζήτηση προβλημάτων. Παρόλο που είναι εφικτή η εξ'αποστάσεως εργασία, καθυστερεί πολύ την ανάπτυξη (αυτό έγινε εμφανές τα τελευταία χρόνια με την απαγόρευση κυκλοφορίας λόγω της πανδημίας)
β) Λογικά βοηθάει στην ροή των συναντήσεων ώστε όλοι οι φοιτητές να βρίσκονται στο ίδιο στάδιο και να μπορούν να ανταλλάσουν απόψεις για τα ίδια προβλήματα, όπως δηλαδή θα συνέβαινε σε ένα πραγματικό σενάριο ανάπτυξης λογισμικού
γ) -> α

> λειτουργείτε μέσα στο σύστημα που σας διατίθεται από το ελληνικό σύστημα εκπαίδευσης με αποτέλεσμα, να δημιουργείται ένα mix μεταξύ της φιλοσοφίας σας και του υπάρχοντος συστήματος, που κατά την δική μου άποψη δεν λειτουργεί προς όφελος ούτε της, κατά τα άλλα ενδιαφέρουσας, φιλοσοφίας του μαθήματος αλλά ούτε και των φοιτητών

Παρ' όλα αυτά συμφωνώ με το παραπάνω, ιδιαίτερα φέτος που έγιναν σημαντικές αλλαγές στα παραδοτέα. Η φιλοσοφία του μαθήματος περιορίζεται από το σύστημα στο οποίο βρίσκεται (ένα απλό παράδειγμα για όσους ήταν μεγαλύτερο έτος είναι οι διαθέσιμες ώρες συνάντησης πάνω στις οποίες βρίσκονται άλλα μαθήματα που έπρεπε να παρακολουθήσουμε).

**Reply by IsabellaPap** [reply]
Ευχαριστώ για την απάντηση σου! Δεν διαφωνώ για τα (α) και (β), απλά εφόσον είναι τόσο ξεκάθαρα για το όφελος του φοιτητή (feedback & ομαδικότητα), δεν θεωρώ πως είναι αναγκαίο να βαθμολογούνται τόσο αρνητικά (0% βαθμολογίας για εκπρόθεσμα παραδοτέα). Ο φοιτητής είναι υπεύθυνος να οργανώνει το διάβασμά του για τα καλύτερα αποτελέσματα, δεν χρειάζεται να επιβάλλεται από τον καθηγητή με τον τρόπο που συμβαίνει τώρα. 

Αν υπήρχε ευελιξία στις προθεσμίες (τουλάχιστον για τον Σεπτέμβρη) πιστεύω (και το είχα αναφέρει και στα συμπεράσματα της αναφοράς μου πέρυσι) πως θα βλέπαμε πολύ πιο ωραίες δουλειές από τους φοιτητές. Αυτό το εβδομαδιαίο κυνηγητό με τις προθεσμίες δεν ωφελεί την ποιότητα των παραδοτέων, ούτε τη σωστή ομαδικότητα (τα περισσότερα post στα discussion των οργανισμών είναι ένας φοιτητής να εξηγεί στους υπόλοιπους πως να κάνουν το παραδοτέο για να προλάβουν). Δεν υπάρχει περιθώριο χρόνου για feedback μεταξύ φοιτητών (με νόημα και όχι απλά για τον βαθμό που προσδίδει η ομαδικότητα).
Αν η φιλοσοφία του μαθήματος είναι να κυνηγάμε την γνώση και την εμπειρία αντί για την βαθμολογία, σε αυτό έχει αποτύχει πλήρως. Ο λόγος, κατά την άποψη μου, είναι το πόσο εύκολο είναι να κοπείς στο μάθημα, πράγμα που κάνει τους φοιτητές να κοιτούν πως θα καταφέρουν να φτάσουν αυτό το άπιαστο 5 εντός του περιορισμένου χρόνου που τους δίνεται, και όχι να δώσουν βάση στο περιεχόμενο του μαθήματος.

Θα βάλω την παραπάνω απάντηση ως answer γιατί από ότι φαίνεται ό,τι προβλήματα έχω με την αυστηρή διαδικασία βαθμολόγησης δεν αφορούν το μάθημα. Ευχαριστώ για τον χρόνο που μου αφιερώσατε.

**Reply by epidrome** [reply]
@IsabellaPap φυσικά σε θυμάμαι (όπως θυμάμαι με τα μικρά τους ονόματα όλους τους φοιτητές που έρχονται τακτικά) και πράγματι είσαι πολύ επιμελής, σίγουρα ανάμεσα στους καλύτερους φοιτητές

Η ευκολία-δυσκολία ενός μαθήματος είναι σχετική με τα άλλα μαθήματα αλλά υπάρχει και ένα αντικειμενικό κριτήριο που είναι ο φόρτος εργασίας, ο οποίος σύμφωνα με τις μονάδες ECTS για αυτό το μάθημα θα πρέπει να είναι χονδρικά κάπου 10 ώρες την εβδομάδα για κάποιον που θέλει να πάρει 10.

> δεν είχα την δυνατότητα να αφιερώνω 4-5 ώρες την εβδομάδα (χωρίς να μετράω τις δια ζώσης συναντήσεις) για τα ζητούμενα του μαθήματός σας

Με αυτόν τον χρόνο (4-5 ώρες/εβδομάδα) ο βαθμός, σύμφωνα με το ECTS θα πρέπει να είναι κάπου πέντε. 

> παρότι ίσως δεν σας ταιριάζει, λειτουργείτε μέσα στο σύστημα που σας διατίθεται από το ελληνικό σύστημα εκπαίδευσης με αποτέλεσμα, να δημιουργείται ένα mix μεταξύ της φιλοσοφίας σας και του υπάρχοντος συστήματος, που κατά την δική μου άποψη δεν λειτουργεί προς όφελος ούτε της, κατά τα άλλα ενδιαφέρουσας, φιλοσοφίας του μαθήματος αλλά ούτε και των φοιτητών (γεγονός που αποδεικνύεται από κάποια στατιστικά επιτυχίας + τα μέτρα που πάρθηκαν φέτος για την διχοτόμηση του μαθήματος σας σε 2 μέρη με άλλο καθηγητή στην επίβλεψη).

Εδώ νομίζω πως μπερδεύεις την εκπαιδευτική φιλοσοφία με τον αντικειμενικό βαθμό δυσκολίας σύμφωνα με το ECTS. Αν λοιπόν σε κάποια άλλα μαθήματα με τα ίδια ECTS και με την ίδια προσπάθεια ο βαθμός σας είναι 9-10 τότε αυτά τα μαθήματα δεν είναι κανονικά αλλά στρεβλώνουν το σύστημα ECTS για να περάσουν περισσότεροι φοιτητές και αυτό το γεγονός που εκτός από εσάς επιβεβαιώνουν και οι στατιστικές που αναφέρετε σημαίνει ότι το πτυχίο που λαμβάνετε δεν έχει την ίδια αξία με άλλα και δίνεται σε πολλούς χαριστικά. Ελπίζω να συμφωνείτε ότι αυτό δεν είναι θέμα εκπαιδευτικής φιλοσοφίας.
